src/error.ts

```diff
@@ -55,4 +55,11 @@ class ParsingError extends ErrorMixin {}
 
 class InputValidationError extends ErrorMixin {}
 
-export { ParsingError, ErrorMixin, InputValidationError };
+class NotImplementedError extends ErrorMixin {}
```

Comment:
where is this error used?

Reply:
As discussed in RC, I initially used it for inserted fusion sequences like "NM_002354.2:r.?_555::guaugauuuuuuaauaa::NM_000251.2:r.212_?" unitl I implemented it later on.
I decieded to keep the class since it has also been implemented elsewhere like the in API (https://github.com/bcgsc/pori_graphkb_api/blob/46c8abe788a899cffc74b406006132b8fe1738dd/src/repo/error.js#L18) so maybe in the future it could be imported by the API at the same time it's importing the parser.